ui/src/components/SampleQueue/CharacterisationTaskItem.jsx

```diff
@@ -74,121 +67,107 @@ function CharacterisationTaskItem(props) {
       pointIDString={pointIDString()}
     >
       <div className={styles.taskBody}>
-        {wedges.map((wedge, i) => {
-          const { parameters } = wedge;
-          const { fileName, path = '' } = parameters;
-          const pathEndPart = path.slice(-40);
-
-          return (
-            <div key={`wedge-${i}`}>
```

Comment:
With whitespace hidden in the diff, you can see that this `div` goes away and the `style` attribute below (`0.5em` is already the default padding in `styles.dataPath`).